src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,17 @@
+package duke.task;
+
+public class Event extends Task {
+    protected String from;
+    protected String to;
```

Comment:
Maybe this could be named as 
```suggestion
    protected String fromDate;
    protected String toDate;
```
Might this be better in helping you remember the variables?